Plugins/NativeXr/Source/NativeXr.cpp

```diff
@@ -2638,6 +2640,7 @@ namespace Babylon
 
             Napi::Reference<Napi::Array> m_jsInputSources{};
             std::map<xr::System::Session::Frame::InputSource::Identifier, Napi::ObjectReference> m_idToInputSource{};
+            std::pair<bool, Napi::ObjectReference> m_eyeTrackedSource{};
```

Comment:
>bool [](http://example.com/codeflow?start=22&length=4)

does this bool serve any purpose? why is it used over a null check for the ObjectReference?

Reply:
if the bool is needed, please create your own struct definition for the type thats being used here compared to relying on a pair with a .first .second fields that don't reflect anything about how these properties are used.